conda/meta.yaml

```diff
@@ -1,22 +1,27 @@
+{% set name = "zstash" %}
+{% set version = "0.4.0" %}
+
 package:
-    name: zstash
-    version: 0.4.0
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-    git_url: https://github.com/E3SM-Project/zstash
-    git_rev: v0.4.0
+  git_url: https://github.com/E3SM-Project/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 9fce7561efa0d512567f89219a1f1642210810523a550b1974a17fdba1b6f544
```

Comment:
Better, faster and safer to use the tar file than the git tag.  This way, if the file changes you find out that the hash is "wrong".